shell/pages/c/_cluster/uiplugins/SetupUIPlugins.vue

```diff
@@ -43,7 +49,7 @@ export default {
       this.installCharts = found;
 
       if (this.$store.getters['management/schemaFor'](CATALOG.CLUSTER_REPO)) {
-        await this.$store.dispatch('management/findAll', { type: CATALOG.CLUSTER_REPO });
+        await this.$store.dispatch('management/findAll', { type: CATALOG.CLUSTER_REPO, opt: { force: true } });
```

Comment:
I noticed we're doing force: true here and elsewhere. It might be reasonable to comment why we need to do a full request in these areas.